test/integration/three_d_secure_ft_tests.js

```diff
@@ -68,16 +67,45 @@ describe('chargeTests', function () {
   })
 
   describe('The /card_details/charge_id/3ds_required', function () {
-    beforeEach(function () {
-      nock.cleanAll()
-    })
     describe('When invoked on a worldpay gateway account', function () {
       it('should return the data needed for the iframe UI', function (done) {
-        const chargeResponse = helper.rawSuccessfulGetCharge(State.AUTH_3DS_REQUIRED, 'http://www.example.com/service', chargeId, gatewayAccountId,
-          {
-            'paRequest': 'aPaRequest',
-            'issuerUrl': 'http://issuerUrl.com'
+        const paRequest = 'aPaRequest'
+        const issuerUrl = 'http://issuerUrl.com'
```

Comment:
I know this was already there but could you change this an unregisterable domain (e.g. issuerUrl.test)?